node/lib/Logger.ts

```diff
@@ -20,62 +22,89 @@ export class Logger implements ILogger {
             this.globalLogLevel = logLevel;
         }
 
-        let consoleTransport = new winston.transports.Console({
+        this.consoleTransport = new winston.transports.Console({
```

Comment:
I agree -- now that we have custom transports being configured. 

---
In reply to: [105340367](https://github.com/openT2T/opent2t/pull/38#discussion_r105340367) [](ancestors = 105340367)